### PR TITLE
Create examples detailing how to use custom callbacks, synthesizers, and models with PrivacyRaven's attack suite

### PR DESCRIPTION
**Pull request by pwang00**
Addresses #47

### CODE REVIEW

**Review by suhacker1 — CHANGES_REQUESTED** [review]
Change "Pytorch Lightning bolt" to "[PyTorch Lightning Bolts](https://pytorch-lightning-bolts.readthedocs.io/en/latest/)". 
 
Immediately linking to the documentation makes it easier for the user to understand the code and eventually implement their own version.

**Comment by suhacker1** [comment]
I'm a little confused as to what the create_model.py file is for since you're not actually creating a new model there. For model creation, I was assuming a new model would be created and then either attacked or used as the substitute model architecture

**Comment by pwang00** [comment]
I believe all changes have been accounted for.  I removed create_model.py since it doesn't seem to do what its filename suggests, but will consider creating a working example and submit it in a future PR.

File: examples/create_synthesizer.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+# User-defined query function
+def query_mnist(input_data):
+    return get_target(model, input_data)
+
+# Obtain seed (or public) data to be used in extraction
+emnist_train, emnist_test = get_emnist_data()
+
+# Create a query function for a target PyTorch Lightning model
+model = train_mnist_victim(gpus=1)
```
[the remark this answers is not in the document]
Comment (pwang00) [reply examples/create_synthesizer.py]: Will do.  Should we also consider make this explicit in PrivacyRaven's README?

[the remark this answers is not in the document]
Comment (suhacker1) [reply examples/create_synthesizer.py]: I think it's okay for now- we don't want to overload the information in the README either

**Review by suhacker1 — APPROVED** [review]

**Pull request merged**